content/en/apps/tutorials/localizing-cht.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "CHT Localization/"
+linkTitle: "cht-localization/"
+weight: 1
+description: >
+  Setting up a local environment to build and test CHT applications
+relatedContent: >
+  core/overview/translations
+  apps/reference/translations
+---
+
+{{% pageinfo %}}
+Given that CHT apps are used around the world, the Core Framework was designed with localization in mind. The Core Framework itself is available in English, French, Hindi, Nepali, Spanish, Swahili, and Indonesian.
+
+This tutorial will take you through localizing the language of CHT applications on CHT version 3.9.1. This includes setting up the instance text as well as outgoing texts to the desired language.
+
+By the end of the tutorial you should be able to:
+
+- Change the CHT instance text to another language; in our case: Swahili
+- Change outgoing texts to another language: Swahili in our case.
+{{% /pageinfo %}}
+
+## Brief Overview of Key Concepts
+
+*Localization* this is  setting up the desired language in CHT for the end user.
+
+*Translations* this is manually setting up extra translations of instance tabs texts or outgoing SMS text. See an outline of how to do that [here](https://docs.communityhealthtoolkit.org/apps/reference/translations/#translations).
+
+## Required Resources
+
+You should have a functioning [CHT instance with `cht-conf` installed locally]({{< ref "apps/tutorials/local-setup" >}}), completed a [project folder]({{< ref "apps/tutorials/local-setup#3-create-and-upload-a-blank-project" >}}) setup, and an [messages-sw.properties]({{< ref "apps/reference/translations" >}}) file.
+
+## Implementation Steps
+
+Create a new file in the 'translations/' folder called messages-sw.properties
+
+1. Localize CHT Elements to Swahili
+_**CHT Instance text**_ - for non-admin users, this is the text that falls under **Messages**, **Tasks**, **Reports**, **People** and **Targets**.
+To localize instance text to Swahili, change the default system language to `Swahili`.
+
+![configuration](change-system-language.png)
```

Comment:
Consider adding step by step screenshots that correspond to the text instructions.